daemon/container_operations_solaris.go

```diff
@@ -15,12 +19,40 @@ func (daemon *Daemon) setupLinkedContainers(container *container.Container) ([]s
 
 // ConnectToNetwork connects a container to a network
 func (daemon *Daemon) ConnectToNetwork(container *container.Container, idOrName string, endpointConfig *networktypes.EndpointSettings) error {
-	return fmt.Errorf("Solaris does not support connecting a running container to a network")
+	return nil
 }
 
 // getSize returns real size & virtual size
 func (daemon *Daemon) getSize(container *container.Container) (int64, int64) {
-	return 0, 0
+	var (
```

Comment:
Agreed. It'll be easier to track once this PR gets in. Every time I rebase the PR there are some scenarios such as this one. I do my due diligence but sometimes.. :)